modules/cactus-web/src/index.ts

```diff
@@ -54,3 +54,4 @@ export { default as useId } from './helpers/useId'
 export { ConfirmModal } from './ConfirmModal/ConfirmModal'
 export { Tag } from './Tag/Tag'
 export { Footer } from './Footer/Footer'
+export { BrandBar } from './BrandBar/BrandBar'
```

Comment:
For a lot of our components this doesn't matter, but I think it'd be better if we got in the habit of writing our exports like this.
```suggestion
export { default as BrandBar } from './BrandBar/BrandBar'
```